docs/dev/annual_updates.rst

```diff
@@ -169,20 +172,27 @@ utilities later.
 
 4. Update Table & Column Transformations
 ----------------------------------------
+We're in the process of transitioning from our old transform framework to a newer, more
+standardized transform framework. Currently, the EIA tables utilize the old transform
+framework and the FERC Form 1 tables utilize the new transform framework.
 
-A. EIA Forms 860/860m/861/923
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+A. EIA Forms
+^^^^^^^^^^^^
 **4.A.1)** Use the EIA ETL Debugging notebook mentioned above to run the initial
 transform step on all tables of the new year of data and debug any failures. If any new
 tables were added in the new year of data you will need to add a new transform function
 for the corresponding dataframe. If new columns have been added, they should also be
 inspected for cleanup.
```

Comment:
This will eventually change, but I think it's fine to leave it like this for now.